src/windows/FileUtils.cpp

```diff
@@ -56,7 +56,7 @@ namespace OS
   bool GetProgramData(std::string& programData)
   {
     LPWSTR wszPath = NULL;
-    if (SHGetKnownFolderPath(FOLDERID_ProgramData, 0, NULL, &wszPath) != S_OK)
+    if (SHGetKnownFolderPath(FOLDERID_ProgramData, 0, NULL, &wszPath) != true)
```

Comment:
Wrong.  SHGetKnownFolderPath is a Windows function and it does not return a bool.

Reply:
Will revert this.